README.md

```diff
@@ -59,3 +59,14 @@ Logs that are specific for SQL Connector:
   - stepName: `import` - showing progress for triggering imports on the platform
 
 * `incoming.job.query` includes the full query which is run on the external database to import data from
+
+### Status
```

Comment:
@mkawalek for the sake of completness can we also add a `Notifications` section here to document `/notifications` calls?